apispec/officer-delta-spec.yml

```diff
@@ -0,0 +1,162 @@
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/Officer_delta'
+  responses:
+    '200':
+      description: Successfully produced message onto Kafka topic.
+    '400':
+      description: Bad request body - validation errors.
+    '401':
+      description: Unauthorised - missing api key in header.
+
+components:
+  schemas:
+    Officer_delta:
+      type: object
+      properties:
+        officers:
+          type: array
+          items:
+            $ref: '#/components/schemas/Officer'
+        CreatedTime:
+          type: string
+        delta_at:
+          type: string
+      required:
+        - officers
+        - CreatedTime
+
+    Officer:
+      type: object
+      properties:
+        company_number:
+          type: string
+        changed_at:
+          type: string
+        kind:
+          type: string
+        internal_id:
+          type: string
+        appointment_date:
+          type: string
+        title:
+          type: string
+          maximum: 50
+        corporate_ind:
+          type: string
+          enum:
+            - Y
+            - N
+        surname:
+          type: string
+          maximum: 160
+        forename:
+          type: string
+          maximum: 50
+        middle_name:
+          type: string
+          maximum: 50
+        date_of_birth:
+          type: string
+        service_address_same_as_registered_address:
+          type: string
+          enum:
+            - Y
+            - N
+        nationality:
+          type: string
+          maximum: 50
+        occupation:
+          type: string
+          maximum: 100
+        officer_id:
+          type: string
+          maximum: 10
+        secure_director:
+          type: string
+          enum:
+            - Y
+            - N
+        officer_detail_id:
+          type: string
+          maximum: 10
+        officer_role:
+          type: string
+        ususal_residential_country:
+          type: string
+          maximum: 160
+        previous_name_array:
+          $ref: '#/components/schemas/Previous_name_array'
+        identification:
+          $ref: '#/components/schemas/Identification'
+        service_address:
+          $ref: '#/components/schemas/Address'
+        usual_residential_address:
+          $ref: '#/components/schemas/Address'
+      required:
+        - corporate_ind
+        - appointment_date
+        - service_address_same_as_registered_address
+        - officer_id
+        - officer_detail_id
+        - secure_director
+
+    Previous_name_array:
+      type: object
+      properties:
+        previous_surname:
+          type: string
+        previous_forename:
+          type: string
+        previous_timestamp:
+          type: string
+
+    Identification:
+      type: object
+      properties:
+        EEA:
```

Comment:
The Identification has ""EEA"/"non_eea"/"UK_limited_company"/"other_corporate_body_or_firm" based on mock delta schemas- https://companieshouse.atlassian.net/wiki/spaces/DataS/pages/3229745153/MOCK+Delta+Schema+Officer+URA+Validation